spec/requests/user_can_send_request_to_community_spec.rb

```diff
@@ -0,0 +1,36 @@
+RSpec.describe 'GET /communities' do
+  let!(:community) { create(:community, code: '2tUnJLGwPYJq') }
+  let!(:community2) { create(:community, code: 'paYljxCDcjGE') }
+  let!(:community3) { create(:community, code: 'Hqxx5Uk5evmD') }
+  
```

Comment:
Some have spaces between blocks and some does not, just a question of consistency. /Kf